src/pyproject_hooks/_impl.py

```diff
@@ -95,13 +95,21 @@ def quiet_subprocess_runner(
 ) -> None:
     """Call the subprocess while suppressing output.
 
-    This uses :func:`subprocess.check_output` under the hood.
+    This uses :func:`subprocess.run` with `PIPE` under the hood.
     """
     env = os.environ.copy()
     if extra_environ:
         env.update(extra_environ)
 
-    check_output(cmd, cwd=cwd, env=env, stderr=STDOUT)
+    subprocess.run(
+        cmd,
+        cwd=cwd,
+        env=env,
+        stdout=subprocess.PIPE,
+        stderr=subprocess.STDOUT,
+        check=True,
+        text=True,
```

Comment:
Using `text=True` would be a behaviour change.

Reply:
Let's drop this line, to be cautious.

Reply:
```suggestion
```